src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandler.java

```diff
@@ -172,9 +173,31 @@ protected void handleAudioVideoDocuments(SscsCaseData sscsCaseData) {
         validateDwpResponseDocuments(sscsCaseData, preSubmitCallbackResponse);
 
         validateDwpAudioVideoEvidence(sscsCaseData, preSubmitCallbackResponse);
+
+        validateSscs5CaseConfidentialityNoAppendix12Doc(sscsCaseData, preSubmitCallbackResponse);
+
         return preSubmitCallbackResponse;
     }
 
+
+    private void validateSscs5CaseConfidentialityNoAppendix12Doc(SscsCaseData sscsCaseData, PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse) {
+        if (Benefit.TAX_CREDIT.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())
```

Comment:
If you get the `Benefit` object, you can just call `e.getSscsType().equals(SscsType.SSCS5)` which is a lot tidier than what you have here and will future proof it in case we add any more.